test/Concurrency/concurrent_value_inference.swift

```diff
@@ -10,7 +10,7 @@ struct S1 {
   var c: C2
 }
 
-enum E1 {
+enum E1 { // expected-note{{add conformance of enum 'E1' to the 'Sendable' protocol}}
```

Comment:
This message is a bit strange. We don’t add conformance _of_ types _to_ protocols. Might I suggest:

> add conformance to the ‘Sendable’ protocol to the enum ‘E1’.

or even

> add ‘Sendable’ conformance to the enum ‘E1’.

Reply:
We use the "consider" wording elsewhere, so I'm going to try out

```
consider making enum 'E1' conform to the 'Sendable' protocol 
```